src/client/src/core/util/validation/rules.ts

```diff
@@ -120,6 +120,14 @@ export const digitsOnly = function(value: string): boolean | string {
   );
 };
 
+export const numericOnly = function(value: string): boolean | string {
```

Comment:
this function looks same as `digitsOnly()` ? why not update it directly

Reply:
yes, need to confirm whether we are going to update the digitsOnly with numericOnly or using a separate rule